packages/core/admin/admin/src/content-manager/hooks/useLazyComponents/index.js

```diff
@@ -36,23 +39,19 @@ const useLazyComponents = (componentUids = []) => {
       setStore(Object.fromEntries(componentStore));
     };
 
-    if (componentUids.length && loading) {
+    if (loading) {
       /**
        * These uids are not in the component store therefore we need to get the components
        */
-      const newUids = componentUids.filter((uid) => !componentStore.get(uid));
-
       const componentPromises = newUids.map((uid) => {
         const customField = customFieldsRegistry.get(uid);
 
         return customField.components.Input();
       });
 
-      if (componentPromises.length > 0) {
```

Comment:
Now we are getting to this code block only when there is something to load so no need to check if there is promises to resolve.